libraries/consul_installation_binary.rb

```diff
@@ -86,11 +87,14 @@ def consul_program
       end
 
       def self.binary_basename(node, resource)
+        new_filename = node['consul']['enterprise'] ? 'consul-enterprise' : 'consul'
+        new_version = node['consul']['enterprise'] ? "#{resource.version}%2bent" : resource.version
```

Comment:
Same here for `new_version` vs. `version`

Reply:
I can use the shorter nomenclature.

Reply:
It has been updated.